From 5612a505cddcdb831f8d4337742b5c4115d0b148 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Wed, 4 Feb 2015 16:18:26 +0800
Subject: [PATCH 1371/1594] MLK-10202-1 ARM: imx6sx-sdb: add tx-d-cal property
 for usbphy

commit a944c36f13b6a9d3eca073516416255e9160210e from
git://git.freescale.com/imx/linux-2.6-imx.git

It is used to improve signal quality for board.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dtsi |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dtsi b/arch/arm/boot/dts/imx6sx-sdb.dtsi
index e3bdbfb..d3f59e6 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dtsi
+++ b/arch/arm/boot/dts/imx6sx-sdb.dtsi
@@ -464,6 +464,14 @@
 	status = "okay";
 };
 
+&usbphy1 {
+	tx-d-cal = <0x5>;
+};
+
+&usbphy2 {
+	tx-d-cal = <0x5>;
+};
+
 &usdhc2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc2>;
-- 
1.7.5.4

